packages/documentation/src/stories/components/icon/icon.stories.tsx

```diff
@@ -0,0 +1,306 @@
+    'base': args.base || null,
+    'flip-h': args['flip-h'],
+    'flip-v': args['flip-v'],
+    'scale': args.scale !== 1 ? args.scale : null,
+    'rotate': args.rotate !== 0 ? args.rotate : null,
+    'animation': args.animation !== 'null' ? args.animation : null,
+  });
+}
+
+const Template = (args: Args) => <post-icon {...normalizeArgs(args)} />;
+
+export const Default: Story = Template.bind({});
+Default.decorators = [
+  (Story: Story) => (
+    <div style={{ fontSize: '32px' }}>
+      <Story />
+    </div>
+  ),
+];
+
+const ColorTemplate = (args: Args) => (
+  <>
+    {forEach(
+      [
+        { class: 'text-primary' },
+        { class: 'text-info' },
+        { class: 'text-success' },
+        { style: { color: 'blue' } },
+        { style: { color: '#800080' } },
+        { style: { color: 'rgb(230, 0, 0)' } },
+      ],
+      (data: { key: number; value: any }) =>
+        Template({
+          ...args,
+          key: data.key,
+          name: '1022',
+          ...data.value,
+        }),
+    )}
+  </>
+);
+
+export const Color: Story = ColorTemplate.bind({});
+Color.decorators = [
+  (Story: Story) => (
+    <div className="d-flex flex-wrap gap-2" style={{ fontSize: '32px' }}>
+      <Story />
+    </div>
+  ),
+];
+
+const SizeTemplate = (args: Args) => (
+  <>
+    {forEach(
+      [
+        {},
+        { class: 'h3' },
+        { class: 'h1' },
+        { class: 'fs-huge' },
+        { style: { fontSize: '4rem' } },
+        { style: { fontSize: '6rem' } },
+      ],
+      (data: { key: number; value: any }) =>
+        Template({
+          ...args,
+          key: data.key,
+          name: '1022',
+          ...data.value,
+        }),
+    )}
+  </>
+);
+
+export const Size: Story = SizeTemplate.bind({});
+Size.decorators = [
+  Story => (
+    <div className="d-flex flex-column">
+      <Story />
+    </div>
+  ),
+];
+
+const StyleTemplate = (args: Args) => (
+  <>
+    {forEach(
+      [
+        { class: 'border rounded p-3', style: { fontSize: '4rem' } },
+        { class: 'border border-success rounded p-3 text-success', style: { fontSize: '4rem' } },
+        { class: 'rounded-circle bg-info p-3', style: { fontSize: '4rem' } },
+      ],
+      (data: { key: number; value: any }) =>
+        Template({
+          ...args,
+          key: data.key,
+          name: '1022',
+          ...data.value,
+        }),
+    )}
+  </>
+);
+
+export const Style: Story = StyleTemplate.bind({});
+Style.decorators = [
+  Story => (
+    <div className="d-flex flex-wrap gap-2">
+      <Story />
+    </div>
+  ),
+];
+
+const FlipTemplate = (args: Args) => (
+  <>
+    {forEach(
+      [{}, { 'flip-h': true }, { 'flip-v': true }, { 'flip-h': true, 'flip-v': true }],
+      (data: { key: number; value: any }) =>
+        Template({
+          ...args,
+          key: data.key,
+          name: '1022',
+          ...data.value,
+        }),
+    )}
+  </>
+);
+
+export const Flip: Story = FlipTemplate.bind({});
```

Comment:
Can you exclude the "flip-h" and "flip-v" properties from this story? It does not work properly. 
Same for the "scale" property in the Scale story, "rotate" property in the Rotate story and so on.

Reply:
I updated allmost every story to show only controls that make sense.